explorer/explorer.go

```diff
@@ -160,6 +164,7 @@ func (exp *explorerUI) rootWebsocket(w http.ResponseWriter, r *http.Request) {
 
 				// Write block data to websocket client
 				exp.NewBlockDataMtx.RLock()
+				exp.MempoolMtx.RLock()
```

Comment:
I'd prefer if we had `MempoolInfo` inherit `sync.RWMutex` so we can do `exp.Mempool.Lock()`